quick-start.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Download the node-creator script
+wget https://raw.githubusercontent.com/JellyPool/meter-node-creator/launch_script/node-creator
```

Comment:
- Fix the script location
- Add a local location to `wget` . If the user triggers `quick-start.sh` from somewhere outside of their home directory, the crontab entry won't work.